### PR TITLE
instant-messenger: add privacy concerns warning to Riot

### PR DESCRIPTION
**Pull request by Mikaela**
Closes: #1007
Closes: #1004
Closes: #1023 (supersedes)

See also: #840

### CODE REVIEW

File: _includes/sections/instant-messenger.html [hunk 1]
```diff
@@ -24,7 +24,7 @@ <h1 id="im" class="anchor"><a href="#im"><i class="fas fa-link anchor-icon"></i>
...
+description="Riot.im is a decentralized free-software chatting application based on the <a href\"https://matrix.org/\">Matrix</a> protocol, a recent open protocol for real-time communication offering E2E encryption. It can bridge other communications via others protocols such as IRC too. <a href=\"https://github.com/vector-im/riot-web/issues/6779\"><span class=\"badge badge-warning\" data-toggle=\"tooltip\" title=\"The end-to-end encryption is currently in beta and the mobile client states 'End-to-end encryption is in beta and may not be reliable. You should not yet trust it to secure data.'\">Experimental <i class=\"far fa-question-circle\"></i></a></span> <a href=\"https://gist.github.com/maxidorius/5736fd09c9194b7a6dc03b6b8d7220d0\"<span class=\"badge badge-danger\" data-toggle=\"tooltip\" title=\"Riot sends a lot of data to matrix.org and vector.im with default settings that aren't trivial to change, also with selfhosted homeservers\">Privacy concerns</span></a>"
```
Comment (ggg27) [inline _includes/sections/instant-messenger.html]: What data does Riot send? They have opt-in analytics, is there any other issues?

Comment (Mikaela) [reply _includes/sections/instant-messenger.html]: Did you click the link, https://gist.github.com/maxidorius/5736fd09c9194b7a6dc03b6b8d7220d0 ?

It has many components that Riot phones home to often, some are mentioned "in a small print" that you have to understand in login screen or settings (even if you cannot change them) or follow some rooms actively to understand and some aren't even mentioned anywhere such as the integration server Scalar.

Comment (ggg27) [reply _includes/sections/instant-messenger.html]: Sorry, I did not see this URL.

**Comment by ggg27** [comment]
You know, it might be better to instead of listing Riot list matrix.
You could them change the description to:

> Matrix is a decentralized free-software protocol for real-time communication offering E2E encryption. It can bridge other communications via others protocols such as IRC too[[1](https://matrix.org/docs/guides/types-of-bridging)]. A full list of clients is listed [here](https://matrix.org/clients/). We recommend using a client with E2E support like [Riot](https://www.wikipedia.org/wiki/Riot.im) or [WeeChat](https://www.wikipedia.org/wiki/WeeChat).

[[Matrix Logo](https://www.wikipedia.org/wiki/Matrix_(protocol)#/media/File:Matrix_logo.svg)]

Instead of a warning it might be better to have a notes tag:
`<a href=\"https://gist.github.com/maxidorius/5736fd09c9194b7a6dc03b6b8d7220d0\"<span class=\"badge badge-danger\" data-toggle=\"tooltip\" title=\"Riot sends a lot of data to matrix.org and vector.im with default settings that aren't trivial to change, also with selfhosted homeservers\">Notes</span></a>`

- Just a thought 

____________________

Btw, this whole comment is licensed under CC0.

**Comment by Mikaela** [comment]
> You know, it might be better to instead of listing Riot list matrix.

This is being discussed in https://github.com/privacytoolsIO/privacytools.io/issues/1005.

>> WeeChat.

I have been using WeeChat for years and it's an IRC client, not a Matrix client. There is a third party scripts adding Matrix support, but it's complicated to install as it's not in the official weechat/scripts repository (or wasn't when I last heard) and it has depedencies that are not available in Debian Testing (or had a week or two ago). I also don't think I could get my family members to use WeeChat, so I think it gets closed out of PTIO.

I cannot think of a good client to replace Riot at this time, but I would be open to PRs adding them as worth mentioning.

**Pull request merged**